docs/spec/deprecated-schema-v1.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Update deprecated schema v1 images
+description: Update deprecated schema v1 images
+keywords: registry, on-prem, images, tags, repository, distribution, api, advanced, manifest
+---
+
+Updating an image manifest from schema 1 to schema 2 is done by downloading an
```

Comment:
I feel there's something missing on this page (some introduction sentence explaining that image manifest v2, schema 1 is deprecated.

@dmcgowan should there be a deprecation note on https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-1.md ?